pyfatx/__init__.py

```diff
@@ -148,6 +148,9 @@ def read(self, path: str, offset: int = 0, size: Optional[int] = None) -> bytes:
 		path = self._sanitize_path(path)
 		attr = self.get_attr(path)
 		assert(attr.is_file)
+		if (offset == 0 and attr.file_size == 0):
```

Comment:
Nit: Drop the parenthesis (unnecessary as the conditional fits on one line).

Reply:
Fixed, muscle memory from too much C programming